DP1.00006.001/Collection.methodology.txt

```diff
@@ -1 +1 @@
-Primary - weighing gauge housed within a small double fence intercomparison reference, which is generally located within 0.5 km of the tower infrastructure. Secondary - tipping bucket located on the top of the tower infrastructure. Throughfall - tipping buckets at 4 or 5 soil array locations (absent at short-stature ecosystems). On flat ground the throughfall collection area is between 37.5 and 50 cm aboveground. Bulk precipitation is determined at five- and thirty-minute intervals for primary precipitation and at one- and thirty-minute intervals for secondary and throughfall precipitation.
+Secondary - tipping bucket located on the top of the tower infrastructure. Throughfall - tipping buckets at 4 or 5 soil array locations (absent at short-stature ecosystems). On flat ground the throughfall collection area is between 37.5 and 50 cm aboveground. Bulk precipitation is determined at one- and thirty-minute intervals for secondary and throughfall precipitation.
```

Comment:
We should include a short note explaining the change, I'm thinking a short second paragraph, something like "In all data Releases prior to RELEASE-2026, Primary precipitation (weighing gauge) was also included in DP1.00006.001. Weighing gauge precipitation can now be found in DP1.00044.001."